audtorch/datasets/speech_commands.py

```diff
@@ -0,0 +1,115 @@
+import os
+import random
+
+from .utils import (download_url, extract_archive, safe_path)
+from .base import AudioDataset
+from .utils import safe_path
+
+
+class SpeechCommands(AudioDataset):
+    r"""An audio dataset of spoken words designed to help train and evaluate
+    keyword spotting systems
+
+    Speech Commands V2 publicly available from Google:
+    http://download.tensorflow.org/data/speech_commands_v0.02.tar.gz
+
+    License: CC BY 4.0
+
+    Args:
+        root (str): root directory of data set, where the CSV files are
+            located, e.g. `/data/speech_commands_v0.02`
+        train (bool, optional): Partition the dataset into the training set.
+        `False` returns the test split. Default: True
```

Comment:
Indention before `False` is missing.
Please change `Default: True` to ``Default: `True` ``.